pkgs/development/python-modules/pyphen/default.nix

```diff
@@ -1,14 +1,26 @@
-{ lib, buildPythonPackage, fetchPypi }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+}:
 
 buildPythonPackage rec {
-  pname = "Pyphen";
+  pname = "pyphen";
   version = "0.11.0";
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "e2c3ed82c3a04317df5102addafe89652b0876bc6c6265f5dd4c3efaf02315e8";
   };
 
+  preCheck = ''
+    sed -i '/addopts/d' pyproject.toml
```

Comment:
Why do we need this? I didn't see such change before.

Reply:
It deletes pytest arguments.

https://github.com/Kozea/Pyphen/blob/master/pyproject.toml#L48